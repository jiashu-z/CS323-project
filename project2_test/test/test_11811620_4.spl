struct A
{
  int num;
};

int func1(){
  return 0.1;
}

int func2(struct A a2){
  int b2[10][10];
  return b2;
}

int func3(int a3[9],int b3[9][9], struct A c3[9], struct A d3[9][9]){
  return a3[0];
}

int func4(struct A a4[9]){
  return a4[0];
}

int test_11811620_4()
{
  int a;
  int b[10][10];
  struct A c[10][10];

  func1(1);
  func2(a);
  func3(b[0], b, c[0], c);
  func3(b, b, c, c);
  func3(c, b);
  a();
  return 0;
}