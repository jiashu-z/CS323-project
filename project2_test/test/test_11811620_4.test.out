Error type 8 at Line 7: incompatiable return type
Error type 8 at Line 12: incompatiable return type: function return value should not be array type
Error type 8 at Line 20: incompatiable return type:unmatched struct type
Error type 9 at Line 29: invalid argument number for func1, expect 0, got 1
Error type 9 at Line 30: invalid argument type for func2. expected: struct, actual: int
Error type 9 at Line 32: invalid argument type for func3. expected: array (dimension:1), actual: array (dimension:2)
Error type 9 at Line 33: invalid argument number for func3, expect 4, got 2
Error type 11 at Line 34: invoking non-function variable: a
